content/en/user-guide/clones/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Clones"
+linkTitle: "Clones"
+weight: 16
+description: Get started with Clones in LocalStack for Snowflake
+---
+
+## Introduction
+
+Cloning in Snowflake allows you to create a quick, zero-copy duplicate of an existing database, schema, or table. This feature enables users to replicate data structures and content for testing or development without duplicating the underlying storage.
+
+The Snowflake emulator supports database cloning, enabling you to create quick, zero-copy duplicates of databases, schemas, or tables. Currently, [`CREATE ... CLONE`](https://docs.snowflake.com/en/sql-reference/sql/create-clone) is supported by LocalStack.
```

Comment:
Technically, zero-copy is not true in our case ;) - Snowflake itself performs zero-copy, but in fact under the covers we are implementing the `CLONE` command by copying the underlying database.
```suggestion
The Snowflake emulator supports database cloning, enabling you to create quick duplicates of databases, schemas, or tables. Currently, [`CREATE ... CLONE`](https://docs.snowflake.com/en/sql-reference/sql/create-clone) is supported by LocalStack.
```